Tasks/DockerCompose/dockercompose.ts

```diff
@@ -25,52 +25,41 @@ else {
     authenticationProvider = new GenericAuthenticationTokenProvider(tl.getInput("dockerRegistryEndpoint"));
 }
 
-authenticationProvider.getAuthenticationToken().then(
-    function success(registryAuthenticationToken) {
```

Comment:
Now Authentication Provider is not returning promise its returning authentication token directly, since rest call is not required anymore. sucess and failure method only exists for the promise.